tests/_support/rest-api-tester/assets/js/src/trap-script.js

```diff
@@ -0,0 +1,145 @@
+(function ($, undefined) {
```

Comment:
I just found out one can set PHPStorm to use WordPress standards for PHP and then Code Style > Javascript (or Typescript) > Set from... > Language > PHP et voila!